l10n_it_delivery_note_base/__init__.py

```diff
@@ -1 +1,13 @@
 from . import models
+from odoo import api, SUPERUSER_ID
+
+
+def create_dn_types_init_hook(cr, sequence):
+    """
+    Create DN types and their sequences after installing the module
+    if they're not already exist
+    """
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    companies = env["res.company"].search([])
```

Comment:
Osservazione al volo. Per cortesia controlla che la search fatta su un model partendo da quell'env (SUPERUSER_ID) applichi il domain con `active` `=` True scartando le company non attive / archiviate

Reply:
Grazie Marco, ma il modello `res.company` non ha il campo `active` a meno che mi sfugga qualcosa.
Infatti anche nella dropdown delle aziende non è possibile archiviarle

Reply:
https://github.com/odoo/odoo/blob/4260561aaefc3f26da1e4ec08c695e415ed3e4d8/odoo/addons/base/models/res_company.py#L60

Opps, sono un po' 16-centrico in questi giorni. Comunque, introdotto qui:
https://github.com/odoo/odoo/commit/7d0996bb151f68647b22719eee5ed4a4c35574bb